README.md

```diff
@@ -59,4 +59,12 @@
 - Use paren bracket style delimiters for preg_* functions and the like `preg_match('(^\d+)', $text)`
 - No trailing whitespace on any line
 - Every file must end with a single new line character
+
+- All PHP functions should have a return typehint if it returns a value.
+ 1. If a return typehint causes the line to exceed the maximum line length then the colon indicating a return type hint should stay with the closing parenthesis.
```

Comment:
is this really saying that the colon always stays with that paran?

Reply:
Yes